echo/docs/adr/0002-billing-period-toggle.md

```
@@ -0,0 +1,24 @@
+# Billing period as a request-time choice with admin override capture
+
+## Status
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)